code/matt/python/lab11.py

```diff
@@ -0,0 +1,61 @@
+"""
+Lab 11
+
+version 1 & 2
+rework
+"""
+
+import string
+
+
+items = " "
+location = 0
+values_for_key_lookup = []
+
+
+def lower_alphabet():
+    """Returns lowsercase ascii alphabet"""
+    return list(string.ascii_lowercase)
+
+
+letters = lower_alphabet()
+
+letter_dict = {letters[i]: i + 1 for i in range(0, len(letters), 1)}
+
+rot_letter_dict = letter_dict.copy()
```

Comment:
What is this? Comments can help you explain to others what your code is doing. Python is great because it's mostly self-explanatory, but there are exceptions.